inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/FieldFormatUtils.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.pojo.sort.util;
+
+import org.apache.inlong.manager.common.enums.FieldType;
+import org.apache.inlong.manager.common.util.JsonUtils;
+import org.apache.inlong.manager.pojo.fieldformat.ArrayFormat;
+import org.apache.inlong.manager.pojo.fieldformat.DecimalFormat;
+import org.apache.inlong.manager.pojo.fieldformat.MapFormat;
+import org.apache.inlong.manager.pojo.fieldformat.StructFormat;
+import org.apache.inlong.manager.pojo.fieldformat.StructFormat.Element;
+
+import java.util.List;
+
+public class FieldFormatUtils {
```

Comment:
Please add Java docs for each class, thanks.

Reply:
Class is too simple to add doc